conda_recipe/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "1.0.0" %}
```

Comment:
I would add a *fourth* version number here so that we can make releases of the validator that are always the same major.minor.patch as the openPMD standard but still have a level for improvements and bug fixes within the same standard version.

Should be possible, right?

Reply:
hm, pbr does have troubles with that although PEP-440 supports it...

Reply:
but reported under: https://bugs.launchpad.net/pbr/+bug/1734361

otherwise we can still use vanilla...